### ISSUE TITLE
Order type in boards selection

### ISSUE THREAD

**Issue by gusarg81** [kiswa/TaskBoard#526]
Hi,

Would be nice to have ordered list in boards selection in dashboard. Example, order By Name, By ID, etc (in my case, would be nice ordered by name, but also maybe have an option for this).
For me, having boards ordered by creations only is not good if you have many boards.

Thanks!

**Comment by kiswa** [comment]
Do you mean in Settings, or in Boards? Because you can change the order they're displayed in on the Settings page Board Administration section.

If you meant the dropdown at the top-left of the boards page, then yeah, that could be useful.

**Comment by gusarg81** [comment]
In Boards, the Select list has no order option.

**Comment by kiswa** [comment]
Makes sense. Are you seeing this as another user setting, or as another selection next to the list?
